src/duktape/log.cr

```diff
@@ -8,8 +8,8 @@ require "log"
 require "json"
 
 Log.setup_from_env(
-  level: ENV.fetch("CRYSTAL_LOG_LEVEL", "INFO"),
-  sources: ENV.fetch("CRYSTAL_LOG_SOURCES", "duktape.*"),
+  log_level_env: "CRYSTAL_LOG_LEVEL",
+  default_sources: ENV.fetch("CRYSTAL_LOG_SOURCES", "duktape.*"),
```

Comment:
If we move from `CRYSTAL_LOG_LEVEL` to `LOG_LEVEL`, should we modify the key here to simply: `LOG_SOURCES`?